### PR TITLE
Implement NVML for NVIDIA GPU Stats

### PR DESCRIPTION
**Pull request by skrashevich**
This pull request updates the method for fetching NVIDIA GPU stats, replacing the previous nvidia-smi command line approach with the more efficient NVML library. The changes include adding the py3nvml dependency, modifying the get_nvidia_gpu_stats() function, and adjusting the display of GPU usage in the frontend. This update provides more accurate and efficient GPU monitoring

### CODE REVIEW

**Comment by NickM-27** [comment]
To make this test pass you are going to need to fix it at the test spot, update the mock to mock the nvml functions

**Comment by skrashevich** [comment]
> update the mock to mock the nvml functions

At best case, it will pass only on machines with Nvidia GPU, at worst - nowhere at all

**Comment by NickM-27** [comment]
I don't think the test would be particularly useful anyway given that the NVML library directly reports the values, there is no parsing of data.

File: frigate/util.py [hunk 1]
```diff
@@ -915,44 +916,41 @@ def get_intel_gpu_stats() -> dict[str, str]:
...
+                gpu_mem_util = -1
 
+            results[i] = {
+                "name": nvml.nvmlDeviceGetName(handle),
+                "gpu": gpu_util,
+                "mem": gpu_mem_util,
+            }
+    except:
         return results
+    return results
```
Comment (NickM-27) [inline frigate/util.py:953]:
```suggestion
    except:
        return results
    
    return results
```

**Review by NickM-27 — CHANGES_REQUESTED** [review]

**Comment by NickM-27** [comment]
This PR adds double `%` for other GPU types so that needs to be addressed as well

Screen Shot 2023-05-03 at 07 14 01 AM

**Comment by NickM-27** [comment]
Can you remove the space between the value and the percent symbol of the other GPU stats to match?

**Review by NickM-27 — APPROVED** [review]

**Pull request merged**